testsuite/python/writevtf.py

```diff
@@ -0,0 +1,46 @@
+from __future__ import print_function
```

Comment:
As already mentioned, the testcase should be based on assertions and preferably inherit from the unittest python module. Please have a look at the other testcases in the testsuite.